core/domain/exp_services.py

```diff
@@ -366,20 +366,23 @@ def get_non_private_exploration_summaries():
         exp_models.ExpSummaryModel.get_non_private())
 
 
-def get_top_rated_exploration_summaries():
+def get_top_rated_exploration_summaries(number_of_explorations_to_fetch):
```

Comment:
Let's call this argument name 'limit'. Shorter, and consistent with usages elsewhere in the codebase.